### ISSUE TITLE
Feature Request: Insert semicolon if it is not there

### ISSUE THREAD

**Issue by Dunky13** [sql-formatter-org/sql-formatter#233]
**Describe the Feature**
If the query doesn't have a semicolon, have an option to insert it

**Why do you want this feature?**
For sanity reasons I'd like to have a semicolon after every query. Even if it is the only query to be formatted.

**Comment by nene** [comment]
Yeah, that's definitely a likely feature to be added. Should be fairly simple to implement as well.

Can't promise any timeline. Pull requests are welcome though.
